plugins/worldinterface/src/worldproxy.h

```diff
@@ -197,13 +197,31 @@ class WorldProxy:public GazeboYarpPlugins::WorldInterfaceServer
    * @return true if success, false otherwise
    */
   virtual bool attach(const std::string& id, const std::string& link_name);
+
+  /**
+    * Attach an object to a link of the robot - takes unscoped names as arguments
```

Comment:
@traversaro The normal way is to use the full scoped name 
`/**
   * Attach an object to a link of the robot.
   * @param id string that identifies object in gazebo (returned after creation)
   * @param link_name name of a fully scoped link (e.g. MODEL1::link)
   * @return true if success, false otherwise
   */
  virtual bool attach(const std::string& id, const std::string& link_name);`
 Instead by unscoped I mean the user can specify  object_model and a link belonging to the object. In case of simple objects like a unit_box containing only one link this function is not useful, but attaching it to a robot like iCub is a bit messy if full scoped name has to be used, as in iCub::l_hand::l_hand_base_link. With attachUnscoped function, user can simply use
`attachUnscoped(unit_box. link. iCub, r_hand_base_link )`